Rules4Net/Data/Constraints/IsTodayConstraint.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Rules4Net.Data.Constraints
+{
+    public class IsTodayConstraint : IConstraint
+    {
+        private string _property;
+
+        public IsTodayConstraint(string property)
+        {
+            this._property = property;
+        }
+
+        public virtual bool Evaluate(IDictionary<string, object> data)
+        {
+            if (!data.ContainsKey(_property) || !(data[_property] is DateTime))
+                return false;
+
+            var value = (DateTime)data[_property];
+            return value.Date.Equals(DateTime.Today);
```

Comment:
I think is better if it is in UTC. We cannot garantee that the timezone of the server is the same of the data